api/rest/bundle_handler.go

```diff
@@ -174,7 +177,7 @@ func collectAll(ctx context.Context, done chan<- []string, dataFile io.WriteClos
 			break
 		}
 		summaryReport.WriteString(fmt.Sprintf("[START GET %s]\n", c.Name()))
-		err := collect(ctx, c, zipWriter)
+		err := collect(ctx, c, zipWriter, collectorTimeout)
```

Comment:
I'd rather create a new `WithTimeout` context from `collectorTimeout` here and pass that as first parameter. Otherwise the API looks pretty weird.

Reply:
Fixed